### ISSUE TITLE
Added Telnet Terminal

### ISSUE THREAD

**Issue by wh010ne** [ManlyMarco/RuntimeUnityEditor#6]
Marco, here's a project solution that has all dependencies and actually build out of the gate. I've added a TCP server to the library that you can connect to at 127.0.0.1:1755. Ultimately, I plan to tie this terminal to the C# REPL. But here is a start. You likely know Unity more than I so please expand on it if it suits your idea.

[v.01 RuntimeUnityEditor-w-Telnet-Console.zip](https://github.com/ManlyMarco/RuntimeUnityEditor/files/3512688/RuntimeUnityEditor-w-Telnet-Console.zip)

**Comment by ManlyMarco** [comment]
Thanks, I'll check it out later. Why not fork the repository and do the edits on the fork? It would make it easier to see the changes.

**Comment by ManlyMarco** [comment]
How about implementing a gui that uses IPC to communicate with the editor? You can use the code from here https://github.com/denikson/COM3D2.KinectCapture

Check the shared code for the IPC and custom named pipe. Also have a look at main code for usage.

**Comment by wh010ne** [comment]
Thnx Marco I'll do that. IPC would be a lot cleaner and manageable.

**Comment by wh010ne** [comment]
Basic IPC server added in fork https://github.com/wh010ne/RuntimeUnityEditor

Includes test client. For now just foundational functionality for IPC communication. Ready for tying to other components. Non-blocking, runs on background thread. Asynchronous. Use's native API. no dependencies.

**Comment by wh010ne** [comment]
Brother, I just noticed something strange. Using Bleeding-Edge BepInEx, things seem to be working good, but I just noticed that if you create a new class that inherits MonoBehavior the class isn't loaded or the Unity workflow like Start(), Update(), OnGUI(), aren't being hit. I'm tracing it, but have you any suggestions on the RuntimeEditor or Unity side? On a roll here until I started putting it through tests.
